test/README.md

```diff
@@ -10,15 +10,13 @@ common usage behaviours work on real Linux distros
 To run locally all you need is:
 
 1. Rust nightly
-2. `libelf`
-3. A checkout of [libbpf](https://github.com/libbpf/libbpf)
-4. `cargo install bpf-linker`
-5. `bpftool`
+1. `libelf`
```

Comment:
Is this needed?

Reply:
Yep. IIRC this is due to libbpf's makefile refusing to install the headers unless libelf is present - although I don't think it's technically required.

Reply:
It doesn't refuse, it just logs a warning.

Reply:
```
tamird@pc:~/src/aya/xtask/libbpf$ ls foo
ls: cannot access 'foo': No such file or directory
tamird@pc:~/src/aya/xtask/libbpf$ make -C src INCLUDEDIR=$PWD/foo install_headers
make: Entering directory '/home/tamird/src/aya/xtask/libbpf/src'
Package libelf was not found in the pkg-config search path.
Perhaps you should add the directory containing `libelf.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libelf', required by 'virtual:world', not found
  INSTALL  bpf.h libbpf.h btf.h libbpf_common.h libbpf_legacy.h bpf_helpers.h bpf_helper_defs.h bpf_tracing.h bpf_endian.h bpf_core_read.h skel_internal.h libbpf_version.h usdt.bpf.h
make: Leaving directory '/home/tamird/src/aya/xtask/libbpf/src'
tamird@pc:~/src/aya/xtask/libbpf$ ls foo
total 12K
drwxr-xr-x  3 tamird tamird 4.0K Jul 18 09:07 .
drwxrwxr-x 11 tamird tamird 4.0K Jul 18 09:07 ..
drwxr-xr-x  2 tamird tamird 4.0K Jul 18 09:07 bpf
```

Reply:
Done.